benches/README.md

```diff
@@ -0,0 +1,65 @@
+# Benchmarks
+
+This directory contains benchmarks for `piccolo`.
+
+## Running benchmarks
+
+```
+cargo bench
+```
+
+Note that it may take a while to compile, because `lto` is set to `fat`.
```

Comment:
Since LTO has been disabled, this should probably be updated.

Reply:
Yep, removed this line. On my machine a compile on a `cargo clean`-ed repository goes from 40s to 20s without `lto=fat`.